docs/users/openmrs-orthanc.md

```diff
@@ -0,0 +1,36 @@
+# OpenMRS-Orthanc Flows
+
+## Flows Overview
+
+``` mermaid
+    sequenceDiagram
+        participant OpenMRS
+        participant Ozone
+        participant Orthanc
+        Orthanc->>Ozone: ImagingStudy
+        Orthanc->>Ozone: Series
+        Orthanc->>Ozone: Instance
+        Ozone->>OpenMRS: Observation (Attachment)
+```
+
+## Flows List
+
+| Source |Element| |Target|         Element          |
+|:------:|:---:|:---:|:---:|:------------------------:|
+|Orthanc |Instance|⭆|OpenMRS| Observation (Attachment) |
```

Comment:
This a list view of the above diagram, therefore the relationship between (Orthanc's) `ImagingStudy` and `Series` and whatever entities in OpenMRS should also appear here.

Reply:
Until now these flows detail the effects of syncing app A's entities on app B's entities, speaking in terms of those apps models. All this to say that in OpenMRS we speak of `Attachment`s (there you are correct) which is a wrapper for an `Obs` (not `Observation`).

Reply:
So made this
```
|Orthanc |Instance|⭆|OpenMRS| Obs (Attachment) |
```